### PR TITLE
Add --mac option support for v2v

### PR DESCRIPTION
**Pull request by xiaodwan**
1) Add support for detecting availability of v2v options
2) Support manually set --mac option in cfg files.
   Ex.
   iface_macs = xx:xx:xx:xx:xx:xx:bridge:br0
   or
   iface_macs = xx:xx:xx:xx:xx:xx:bridge:br0;xx:xx:xx:xx:xx:xx:bridge:br1
3) Automatically set --mac option if available.
4) Rollback to -b/-n option if --mac is not available.

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
Auto detect single mac address
(1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_ovirt.esx.vm.6_7.linux.latest6.arch_x86_64.raw_f.NFS.rhv.it_default: PASS (1900.21 s)
Manually set single mac address
(1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_ovirt.esx.vm.6_7.linux.latest6.arch_x86_64.raw_f.NFS.rhv.it_default: PASS (1879.00 s)
Auto detect multiple mac addresses
(1/1) type_specific.io-github-autotest-libvirt.specific_kvm.positive_test.linux.noyumrepo-rhn.esx.esx_67.output_mode.rhev.rhv.it_vddk: PASS (2416.43 s)
Manully set multiple mac addresses
(1/1) type_specific.io-github-autotest-libvirt.specific_kvm.positive_test.linux.noyumrepo-rhn.esx.esx_67.output_mode.rhev.rhv.it_vddk: PASS (2322.95 s)
xen:
(1/1) type_specific.io-github-autotest-libvirt.function_test_xen.positive_test.rhev.rhel6_2.output_mode.rhev.rhv: PASS (1697.05 s)
libvirt:
(1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.6_7.linux.latest8.arch_x86_64.it_vddk: PASS (1155.55 s)
function_test_esx..esx_uri
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.uefi.ovmf.rhel7.esx_67.rhev.rhv_upload.it_vddk.esx_uri: PASS (2220.64 s)
convert_from_file
(1/1) type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.linux.input_mode.vmx.vmx_ssh.latest8.output_mode.rhev.rhv_upload: PASS (1716.84 s)
input_mode = vmx
(1/1) type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.windows.input_mode.vmx.vmx_default.win2019.output_mode.libvirt: PASS (638.43 s)
The change has no influence on v2v_option
(1/1) type_specific.io-github-autotest-libvirt.v2v_options.positive_test.option_of.from_disk.preallocated.raw_format.input_mode.disk.image.raw_format.sparse.output_mode.libvirt: PASS (122.92 s)
(1/1) type_specific.io-github-autotest-libvirt.v2v_options.positive_test.format_convert.input_mode.libvirtxml.raw_qcow2.output_mode.rhev: PASS (584.70 s)

**Comment by xiaodwan** [comment]
@chloerh @chunfuwen Please help to review this PR, thanks.

File: virttest/utils_v2v.py [hunk 1]
```diff
@@ -1125,3 +1199,80 @@ def v2v_mount(src, dst='v2v_mount_point'):
...
+
+def v2v_supported_option(opt_str):
+    """
+    Check if an cmd option is supported by v2v
+
+    :param opt_str: option string for checking
+    """
+    cmd = 'virt-v2v --help'
+    result = process.run(cmd, verbose=True, ignore_status=True)
+    result.stdout = results_stdout_52lts(result)
```
Comment (chloerh) [inline virttest/utils_v2v.py]: I'm not sure about this, but is it still necessary to use function 'results_stdout_52lts' now?

Comment (xiaodwan) [reply virttest/utils_v2v.py]: I'm not sure, either. But since nobody talked about this, it's better to keep it.

**Pull request merged**